docs/Value-Forms.md

```diff
@@ -156,4 +156,13 @@ When the value say, `Test ©` is supplied as the value for the parameter `exampl
       "identifier": "kebabCase"
     }
 }
-```
\ No newline at end of file
+```
+
+**`snakeCase`** - Converts the value to snake case using the casing rules of the invariant culture. Available since .NET 9.0.100.
```

Comment:
I am open to backporting it to whatever versions are available. Technically it should be able to be backported as far back as `.NET 5.0.300` since I patterned all of my code after the `kebabCase` just above it.

I am not sure how that is done though, and will require some assistance. This is my first contribution to this repo.

Reply:
At this point in time the only real options are:

* 9.0.200 for February's release
* 10.0.100 for next November's release

We don't generally add new features to an already-published release except for very special circumstances. 

Once this PR is merged we'll take care of the back-porting, no worries :)